docs/multiple-tests/pattern-vulnerability/results.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<checkstyle version="1.5">
```

Comment:
is there a chance of the tool eventually reporting more issues? (if more vulnerabilities are found impacting this version)
that might break the tests and the automatic deployment 😞